deepchecks/checks/methodology/unused_features.py

```diff
@@ -64,8 +65,26 @@ def __init__(self, feature_importance_threshold: float = 0.2, feature_variance_t
         self.n_top_unused_to_show = n_top_unused_to_show
         self.random_state = random_state
 
-    def run_logic(self, context: CheckRunContext) -> CheckResult:
-        """Run check.
+    def run(self,
+            train_dataset: Dataset,
+            test_dataset: Dataset,
+            model: BasicModel = None,
+            feature_importance_force_permutation: bool = None,
+            feature_importance_timeout: int = None) -> CheckResult:
+        """Run the check.
+
+        Parameters
+        ----------
+        train_dataset : Dataset
+            dataset representing data an estimator was fitted on
+        test_dataset : Dataset
+            dataset representing data an estimator predicts on
+        model : BasicModel
+            A scikit-learn-compatible fitted estimator instance
+        feature_importance_force_permutation : bool , default None
+            force calculation of permutation features importance
+        feature_importance_timeout : int , default None
```

Comment:
```suggestion
        feature_importance_timeout : int , default: None
```

Reply:
why? all the rest of docstrings doesn't have `:` after the default

Reply:
Where did you check?

Reply:
I saw in Dataset, looks like there are both ways